### ISSUE TITLE
cannot disable publisher font, and it is always bold

### ISSUE THREAD

**Issue by spajak** [kobolabs/epub-spec#49]
I have an ebook (epub, probably 2) where changing the font from embedded to custom is impossible. Also all text is in bold (except italics). 
Clara HD.

**Comment by wareid** [comment]
Can you retest using these steps? https://github.com/kobolabs/epub-spec#sideloading-for-testing-purposes

**Comment by spajak** [comment]
Yes it is working in `kepub`, but italics became bold in custom (not embedded) font Bookerly.

**Comment by wareid** [comment]
When you installed bookerly did you include the italic typeface? I'm wondering if it's defaulting instead to bold.

**Comment by spajak** [comment]
Sure, there are 4 files in `/fonts`:
```
Bookerly-Bold.ttf
Bookerly-BoldItalic.ttf
Bookerly-Italic.ttf
Bookerly-Regular.ttf
```
And it works in all `epub`s, except when loaded as `kepub.epub`.
